content/en/account_management/_index.md

```diff
@@ -66,3 +70,4 @@ You can set your timezone, desktop notifications, and email subscriptions from t
 [10]: /account_management/org_settings
 [11]: /integrations/github
 [12]: https://app.datadoghq.com/account/preferences
+[13]: https://docs.datadoghq.com/help
```

Comment:
```suggestion
[13]: /help
```

No need to enter the full link since we are in the doc website. Having a relative link makes it redirect to the corresponding language doc. Otherwise it would lead to the english version everytime